darts/utils/historical_forecasts/utils.py

```diff
@@ -782,8 +782,7 @@ def _adjust_historical_forecasts_time_index(
     show_warnings: bool,
 ) -> TimeIndex:
     """
-    Shrink the beginning and end of the historical forecasts time index based on the values of `start`,
-    `forecast_horizon` and `overlap_end`.
+    Shrink the beginning and end of the historical forecasts time index based on the value of `start`.
```

Comment:
Perfect, can you please also update the comment above the call the `_adjust_historical_forecasts_time_index()` in `_get_historical_forecast_boundaries` as well?